storage/gcloud/aio/storage/storage.py

```diff
@@ -388,6 +401,25 @@ async def _download(self, bucket: str, object_name: str, *,
 
         return data
 
+    async def _download_stream(self, bucket: str, object_name: str, *,
+                        params: Optional[Dict[str, str]] = None,
+                        headers: Optional[Dict[str, str]] = None,
+                        timeout: int = 10,
+                        session: Optional[Session] = None) -> StreamResponse:
+        # https://cloud.google.com/storage/docs/json_api/#encoding
```

Comment:
Probably meant to use 
```
https://cloud.google.com/storage/docs/request-endpoints#encoding
```

Reply:
yeah IDK this is just copy-pasted from `_download`, I included that comment but never followed the link. It does look broken, so I can fix in both places while I'm under the hood

Reply:
Huh, yeah, looks like they changed the link on us. @jackton1 's link is indeed the one we meant to point to.